src/test/java/org/itsallcode/io/TestCapturingOuputStream.java

```diff
@@ -55,4 +70,39 @@ void testWithBufferedOutputStreamOnTopOfSystemOut() throws IOException
             System.setOut(previousOut);
         }
     }
+
+    @Test
+    void testOutputForwardedWhenCapturing() throws IOException
+    {
+        final OutputStream delegate = mock(OutputStream.class);
+        final CapturingOutputStream stream = new CapturingOutputStream(delegate);
+        stream.capture();
+        final byte[] content = "content".getBytes();
+        stream.write(content);
+        stream.close();
+        verify(delegate).write(content, 0, 7);
```

Comment:
Use length of content as third parameter to express the intent better.

Reply:
✅